exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java

```diff
@@ -869,44 +861,44 @@ protected void processEvent(final StateEvent event) {
          * cause this to be called recursively.
          */
         switch (newState) {
-        case CANCELLATION_REQUESTED: {
```

Comment:
Can you undo spacing changes? `switch` and `case` should be at the same indentation.